consensus/service/src/bin/main.rs

```diff
@@ -42,6 +39,8 @@ fn main() -> Result<(), ConsensusServiceError> {
     let (logger, _global_logger_guard) = create_app_logger(o!(
         "mc.local_node_id" => local_node_id.responder_id.to_string(),
     ));
+    mc_common::setup_panic_handler();
+    let _sentry_guard = mc_common::sentry::init();
```

Comment:
I am a little unsure about this change in particular, for instance, if `Config.parse()` fails, I guess we won't get a sentry alert?

Is that desirable?

Maybe it's desirable to move the fee_map and governer's map stuff after the logging initialization, if there's no reason it has to be before?

Reply:
My thinking on this was that if the process fails to start immediately we should be notified because the pod refuses to initialize. But there's actually no reason not to initialize Sentry before anything else, so I'll make that the first thing on all binaries that use it.